docs/migration_2_to_3.rst

```diff
@@ -26,12 +26,12 @@ As soon as it detects an old-style configuration file, it will convert it to the
 
 Alternatively, you can convert any old configuration file using the conversion tool |convert_config|_:
 
-.. |convert_config| replace:: :obj:`convert-config`
-.. _convert_config: https://github.com/eth-cscs/reframe/blob/master/tools/convert-config
+.. |convert_config| replace:: :obj:`rfm_convert-config`
```

Comment:
```suggestion
.. |convert_config| replace:: :obj:`reframe-convert-config`
```

Reply:
I don't like that a "random" tool will be placed in `bin/`. I would suggest to migrate the functionality of the conversion tool inside ReFrame, by using an additional option, which will only do the conversion and exit.

```
reframe --upgrade-config-file=foo.json
```

Reply:
The documentation about the migration should also be adapted.